packages/api-client/src/api/bundles/bundles.ts

```diff
@@ -99,8 +99,8 @@ export default gql`
                   mediaImages(search: { sort: ASC }) {
                     ...MediaImages
                   }
-                  attributes(filter: $attributeFilters) {
-                    ...Attribute
+                  attributeValues(filter: { isPublic: true, page: $attributesPage, offset: $attributesOffset }) {
```

Comment:
Please leave the `$attributeFilters` as they were before don't hardcode default to `{ isPublic: true, page: $attributesPage, offset: $attributesOffset }`